src/types/granule.graphql

```diff
@@ -150,6 +154,8 @@ type Query {
     conceptId: [String] @deprecated(reason: "Use `params.conceptId`")
     "Cursor that points to the a specific position in a list of requested records."
     cursor: String @deprecated(reason: "Use `params.cursor`")
+    "Search by cycle which is part of the track information of a granule."
+    cycle: Int
```

Comment:
All of the parameters directly on the granule query are deprecated, these values should only be added to GranulesInput